src/ui/components/ToggleSwitch.vue

```diff
@@ -1,23 +1,37 @@
 <template>
-<label class="c-toggle-switch">
-    <input
-        :id="id"
-        type="checkbox"
-        :checked="checked"
-        @change="onUserSelect($event)"
+<div class="c-toggle-switch">
+    <label class="c-toggle-switch__control">
+        <input
+            :id="id"
```

Comment:
What's the id used for in this component?

Reply:
its a checkbox, it always needs an id